backbone/src/main/java/org/researchstack/backbone/step/ToggleFormStep.java

```diff
@@ -0,0 +1,23 @@
+package org.researchstack.backbone.step;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 1/3/17.
+ */
+
+public class ToggleFormStep extends NavigationFormStep {
```

Comment:
because its different in the form of what it's question steps are, only boolean questions, which is controlled by survey factory injecting boolean steps into the constructor